### PR TITLE
[VOL-94] Correct mount in plugin

### PR DESCRIPTION
**Pull request by robhaswell**
https://clusterhq.atlassian.net/browse/VOL-94

Make the plugin tell Docker the correct mountpoint for the volume.

What this test does is fix the current behaviour - that the same path is always passed up. It would be nicer to test for the _correct_ path but I'm not sure that's possible, without making assumptions about the location and structure of `/var/lib/dvol`. Perhaps the reviewer could offer some thoughts?

This branch also contains one unrelated change, which corrects a comment in a test. Sorry.

### CODE REVIEW

**Comment by robhaswell** [comment]
This fails on Docker 1.8. I am investigating.

**Comment by robhaswell** [comment]
Found and fixed.

File: dvol_python/test_plugin.py [hunk 1]
```diff
@@ -269,6 +269,40 @@ def cleanup():
         self.fail("Volume 'docker-volume-list-test' not found in Docker "
                 "output:\n\n" + docker_output)
 
+    def test_unique_volumes(self):
+        """
+        Two separate volumes do not share the same filesystem namespace.
+        """
+        alpha = "test-unique-volume-alpha"
+        beta = "test-unique-volume-beta"
```
Comment (brendan-donegan) [inline dvol_python/test_plugin.py]: These could be capitalised to make it clear they are 'labels'

**Comment by brendan-donegan** [comment]
Minor comment, address (or not) and merge

**Comment by james-w** [comment]
Looks good to me too fwiw.

File: dvol_python/test_plugin.py (hunk 1 above)
Comment (robhaswell) [inline dvol_python/test_plugin.py]: Thanks however I would like to keep the style the same as the rest of the tests.

**Pull request merged**